lib/cssToJss.js

```diff
@@ -58,7 +58,12 @@ function toJssRules(cssRules, options) {
   }
 
   function formatProp(prop) {
-    return options.dashes ? prop : camelCase(prop)
+    if (options.dashes) {
```

Comment:
if dashes is true, it should just return the property as it is, it should only capitalize when it is false

Reply:
Fixed in 859d2a1